src/victory-transition/victory-transition.js

```diff
@@ -93,6 +97,12 @@ export default class VictoryTransition extends React.Component {
     const combinedProps = defaults(
       {domain}, transitionProps, child.props
     );
+
+    if (this.state && this.state.nodesDoneClipPathExit && this.state.nodesWillExit) {
+      const index = props.animationWhitelist.indexOf("clipWidth");
+      props.animationWhitelist.splice(index, 1);
```

Comment:
fixed https://github.com/FormidableLabs/victory-core/pull/92/commits/84c7b7f18108c389a6efa5719808072556ddf3c3 and https://github.com/FormidableLabs/victory-core/pull/92/commits/f550307d22b170b03b47a412669a0ace71212826